src/Rule/BlankLineAfterDirective.php

```diff
@@ -35,6 +35,7 @@ public static function getGroups(): array
     public function check(Lines $lines, int $number): ?string
     {
         $lines->seek($number);
+        \assert($lines->valid());
```

Comment:
This is not needed anymore as `seek` is throwing an exception

Reply:
fixed 👍🏼